examples/tutorials/date_time_charts.py

```diff
@@ -0,0 +1,349 @@
+# that could be passed as region. Additionally, a spacing argument
+# can be passed to increase the range past the maximum and minimum
+# data points.
+
+data = [
+    ["20200712", 1000],
+    ["20200714", 1235],
+    ["20200716", 1336],
+    ["20200719", 1176],
+    ["20200721", 1573],
+    ["20200724", 1893],
+    ["20200729", 1634],
+]
+df = pd.DataFrame(data, columns=["Date", "Score"])
+df.Date = pd.to_datetime(df["Date"], format="%Y%m%d")
+
+fig = pygmt.Figure()
+region = pygmt.info(
+    table=df[["Date", "Score"]], per_column=True, spacing=(700, 700), coltypes="T"
+)
+
+fig.plot(
+    region=region,
+    projection="X15c/10c",
+    frame=["WSen", "afg"],
+    x=df.Date,
+    y=df.Score,
+    style="c0.4c",
+    pen="1p",
+    color="green3",
+)
+
+fig.show()
+
+########################################################################################
+# Setting Primary and Secondary Time Axes
+# ---------------------------------------
+#
+# This example focuses on labeling the axes and setting intervals
+# at which the labels are expected to appear. All of these modifications
+# are added to the ``frame`` parameter and each item in that list modifies
+# a specific section of the plot.
+#
+# Starting off with ``WS``, adding this string means that only
+# Western/Left (**W**) and Southern/Bottom (**S**) borders of
+# the plot will be shown. For more information on this, please
+# refer to :doc:`frame instructions </tutorials/frames>`.
+#
+# The other important item in the ``frame`` list is
+# ``"sxa1Of1D"``. This string modifies the secondary
+# labeling (**s**) of the x-axis (**x**). Specifically,
+# it sets the main annotation and major tick spacing interval
+# to one month (**a1O**) (capital letter o, not zero). Additionally,
+# it sets the minor tick spacing interval to 1 day (**f1D**).
+# The labeling of this axis is also modified using
+# ``pygmt.config(FORMAT_DATE_MAP="o")`` to use the month's
+# name instead of its number.
+
+x = pd.date_range("2013-05-02", periods=10, freq="2D")
+y = [4, 5, 6, 8, 9, 5, 8, 9, 4, 2]
+
+fig = pygmt.Figure()
+with pygmt.config(FORMAT_DATE_MAP="o"):
+    fig.plot(
+        projection="X15c/10c",
+        region=[datetime.datetime(2013, 5, 1), datetime.datetime(2013, 5, 25), 0, 10],
+        frame=["WS", "sxa1Of1D", "pxa5d", "sy+lLength", "pya1+ucm"],
+        x=x,
+        y=y,
+        style="c0.4c",
+        pen="1p",
+        color="green3",
+    )
+
+fig.show()
+
+########################################################################################
+# The same concept shown above can be applied to smaller
+# as well as larger intervals. In this example,
+# data is plotted for different times throughout two days.
+# Primary x-axis labels are modified to repeat every 6 hours
+# and secondary x-axis label repeats every day and shows
+# the day of the week.
+#
+# Another notable mention in this example is
+# ``pygmt.config(FORMAT_CLOCK_MAP="-hhAM")``
+# which specifies the used format for time.
```

Comment:
Similarly, should link to https://docs.generic-mapping-tools.org/latest/gmt.conf.html#term-FORMAT_CLOCK_MAP here:

```suggestion
# setting :gmt-term:`FORMAT_CLOCK_MAP` to "-hhAM"
# which specifies the format used for time.
```